src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.ExpressionCodeGenerator.cs

```diff
@@ -136,32 +133,6 @@ private ExpressionSyntax WrapInCheckedExpressionIfNeeded(ExpressionSyntax expres
                     return SyntaxFactory.CheckedExpression(kind, expression);
                 }
 
-                protected override SyntaxNode GetOutermostCallSiteContainerToProcess(CancellationToken cancellationToken)
```

Comment:
all this logic for determining what container to update was moved up to base type.